chapter09/tests/dags/01_test_dag_integrity.py

```diff
@@ -0,0 +1,24 @@
+"""Test integrity of DAGs."""
```

Comment:
This test is not automaticcaly resolved by pytest. You need to name it `test_something.py`